docs/PORTING.md

```diff
@@ -1,36 +1,43 @@
-# Porting How-To
+# Porting how-to
 
-This document describes the requirements and necessary steps required to port
-`mcuboot` to a new target `OS`.
+This document describes the requirements and necessary steps needed to
+port `MCUboot` to a new target `OS`.
 
 # Requirements
 
-* `mcuboot` requires a configuration file, which can be included as
-   mcuboot_config/mcuboot_config.h, which configures various options
-   (that begin with MCUBOOT_).
-
-* `mcuboot` requires that the target provides a `flash` API with ability to
-  get the flash's minimum write size, and read/write/erase individual sectors.
-
-* `mcuboot` doesn't bundle a cryptographic library, which means the target
-  OS must already have it bundled. The supported libraries at the moment are
-  either `mbed TLS` or the set `tinycrypt` + `mbed TLS` (where `mbed TLS` is
-  used to provide functionality not existing in `tinycrypt`).
+These are the requirements for porting `MCUboot`:
```

Comment:
This is header for the list of requirements, so I think it would be enough to state:
"The requirements for porting `MCUboot`

Reply:
Personally, I prefer to have a full sentence before any colon, as it clarifies the meaning.